python/cugraph/cugraph/sampling/random_walks.py

```diff
@@ -57,7 +57,7 @@ def random_walks(G,
     if max_depth is None:
         raise TypeError("must specify a 'max_depth'")
 
-    G, _ = cugraph.utilities.check_nx_graph(G)
+    G, _ = ensure_cugraph_obj_for_nx(G)
```

Comment:
NetworkX does not have a "random_walks" function, which is why I assume the "isNx" flag is ignored.  But in that cases, shouldn't the whole check also be skipped?

Reply:
offline discussion: if we support Nx types, then our return type should look closer to something Nx would return (Nx does not have a RW API, so we can't copy an API signature for this).
One option is to restrict that API to just cuGraph types, then open it up to Nx when we can later provide a conversion for the return type that is closer to Nx style (ie. on the CPU at least).  Another is do nothing and add a FIXME with these points while we decide.  We decided the FIXME would be best for now.